src/components/remotecontrol/remotecontrol.css

```diff
@@ -102,7 +134,67 @@
     -ms-user-select: none;
 }
 
-@media all and (orientation: portrait) and (max-width: 50em) {
+.contextMenuList {
+    padding: 1.5em 0;
+}
+
+.contextMenuList i.listItemIcon {
+    font-size: x-large;
+}
+
+.nowPlayingPageImageContainer {
+    width: 16%;
+}
+
+.nowPlayingPageImage {
+    margin: 0 auto;
+    width: 100%;
+}
+
+.nowPlayingSecondaryButtons {
+    display: -webkit-box;
+    display: -webkit-flex;
```

Comment:
@MrTimscampi This is already handled by gulp, right?

Reply:
We have yet to remove all these. I wonder if we can enforce this with Stylelint or another tool.

Reply:
You want to remove all the `-moz`, `-webkit`?

Reply:
@samuel9554 Yeah, as that's already handled by gulp. However, wait until @MrTimscampi replies, because he knows more about this. Maybe he's already working on a way to remove all of these at once and it isn't worth the hassle for now :).